app/src/main/res/values/strings.xml

```diff
@@ -708,6 +713,8 @@
     <string name="soundcloud_go_plus_content">This is a SoundCloud Go+ track, at least in your country, so it cannot be streamed or downloaded by NewPipe.</string>
     <string name="private_content">This content is private, so it cannot be streamed or downloaded by NewPipe.</string>
     <string name="youtube_music_premium_content">This video is available only to YouTube Music Premium members, so it cannot be streamed or downloaded by NewPipe.</string>
+    <string name="account_terminated">Account terminated</string>
+    <string name="service_provides_reason">%s provides this reason:</string>
```

Comment:
Maybe some language would put the message in another place
```suggestion
    <string name="service_provides_reason">%1$s provides this reason: %2$s</string>
```

Reply:
That messages is in another TextView with extra margin and italic fonts.

Reply:
Oh right, sorry